src/ensembl/README.md

```diff
@@ -1,38 +1,46 @@
-- `npm run certify` - Runs `setup-ssl.js` to create a local SSL certificate to run the production build on `HTTPS`. There are two files that are created for this: `localhost.crt` and `localhost.key`.
 - `npm run lint` - Runs both `lint:scripts` and `lint:style`. There is no need to run any of these lint scripts during development. This is because, each time a change is made to the code, the linters run along with the build.
 - `npm run lint:scripts` - Runs ESLint to check code errors in TypeScript and React. Also, suggests improvements to the code.
 - `npm run lint:styles` - Does the same as above through `stylelint` for the SASS code.
-- `npm run storybook` - Starts the Storybook application
-- `npm run deploy-docs` — Builds the Storybook application and deploys it, along with the docs on Genome browser, to Github Pages ([link](https://ensembl.github.io/ensembl-client))
 - `npm run check-types` – Runs typescript compiler to check for type correctness
 - `npm jest` - Runs `jest`in non-verbose mode.
 - `npm test` - Runs `jest` to check whether the unit tests pass in verbose mode.
 - `npm test:watch` - Runs `jest` on watch mode.
 - `npm run coverage` - Updates the `jest` coverage of the React.js code, and shows the test coverage.
 - `npm run snapshot` - Updates the `jest` snapshots. You will sometimes need to run this after a change is made to the React.js code. However, first check whether the unit tests are passing after updating them. If you get any errors related to `jest` snapshots, you will then need to run this script.
 
+### Deployment (interal use only)
+- `npm run serve:prod` - This runs the built production site locally using `http`.
+- `npm run serve:prod:secure` - Runs the build production site locally securely using `https`. You will need to run `certify` before running this, in case you already haven't generated an SSL certificate.
+- `npm run build` - Runs the production build. It will initially delete the existing local production build and replace it with the new one.
+- `npm run prod:analyse` — Runs production build, and also uses `webpack-bundle-analyzer` to report the size of the bundle.
+- `npm run certify` - Runs `setup-ssl.js` to create a local SSL certificate to run the production build on `HTTPS`. There are two files that are created for this: `localhost.crt` and `localhost.key`.
+- `npm run deploy` - Runs `deploy.js` file to deploy the production build into the master machine. You will need to pass the full address of the machine name along with your username as an argument.
+- `npm run deploy-docs` — Builds the Storybook application and deploys it, along with the docs on Genome browser, to Github Pages ([link](https://ensembl.github.io/ensembl-client))
+
+## Development
+
+Developers wishing to investigate or contribute to the ensembl-client codebase may also wish to install an IDE. We recommend VS Code, as it has great support for TypeScript and React.js.
 
-### IDE Setup
+### VS Code Extensions
```

Comment:
This section is probably something we can move to our internal docs.

Reply:
If we're going to say anything about IDEs here, then listing useful extensions seems helpful. Otherwise the whole thing should be internal.

Reply:
My vote is for making this whole section internal